ansible/playbooks/roles/compatibility_layer/defaults/main.yml

```diff
@@ -16,34 +17,30 @@ gentoo_prefix_path: /cvmfs/{{ cvmfs_repository }}/versions/{{ eessi_version }}/c
 gentoo_git_repo: https://github.com/gentoo/gentoo.git
 # Select a specific commit in the gentoo_git_repo that should be used for the bootstrap,
 # e.g. by checking: https://github.com/gentoo/gentoo/commits/master
-# April 17 (29492845e41ea6a0a4a9769c7e0ce287d106079b) commit is after fix for Lmod
-# gentoo_git_commit: 29492845e41ea6a0a4a9769c7e0ce287d106079b
-# June 8 (aab8473aa90e0287553b3348a5c5b17872df4b7b) commit that was current when fetching luaposix
-gentoo_git_commit: aab8473aa90e0287553b3348a5c5b17872df4b7b
+# June 6 2025 (6c64b0693572d25ee87ffba030cd345204365f05) commit that includes Lmod 8.7.55 and glibc 2.41
+gentoo_git_commit: 6c64b0693572d25ee87ffba030cd345204365f05
 prefix_required_space: 15 GB
 prefix_user_defined_trusted_dirs:
```

Comment:
Does order matter here? Is the first entry in this list also the first one that's considered (so it is effectively the "override" path)?

Reply:
They are passed in that order to glibc (https://github.com/EESSI/compatibility-layer/blob/main/ansible/playbooks/roles/compatibility_layer/tasks/set_glibc_trusted_dirs.yml#L16), so I would hope/expect that the order matters, but I'll see if I can check that once the build has successfully completed.